arbiter-core/src/tests/signer.rs

```diff
@@ -0,0 +1,31 @@
+use std::sync::Arc;
+
+use super::*;
+const TEST_AGENT_NAME: &str = "test_agent";
+#[test]
+fn simulation_agent_wallet() {
```

Comment:
I think we should drop any use of `Agent` for now. There are a few places I see it still in here. Consider making this change so we can use more precise terminology in the future.